assets/app/views/_pod-template.html

```diff
@@ -72,6 +72,19 @@
         </div>
       </div> 
 
+      <div row ng-if="detailed" ng-repeat="mount in container.volumeMounts" class="icon-row">
+        <div>
+          <!-- For some reason the DB icon is massive compared to the others -->
+          <span aria-hidden="true" style="font-size:13px;line-height:normal" class="fa fa-database"></span>
```

Comment:
Hm, I was able to disable this style checking out your pull request, and it looked OK in Chrome for me.